ledger/participant-integration-api/src/main/scala/platform/store/backend/postgresql/PostgresStorageBackend.scala

```diff
@@ -305,4 +306,27 @@ private[backend] object PostgresStorageBackend
   override def lock(id: Int): DBLockStorageBackend.LockId = PGLockId(id.toLong)
 
   override def dbLockSupported: Boolean = true
+
+  // TODO Cleanup: Refactor into or re-use an existing utility
+  private def tryAcquireConnection(dataSource: DataSource, numberOfAttempts: Int = 600)(implicit
```

Comment:
Why the number of attempts is so high? Is it because of waiting for db availability?